mne/epochs.py

```diff
@@ -249,14 +250,11 @@ def get_data(self):
 
         Returns
         -------
-        data : array of shape [n_epochs, n_channels, n_times]
+        data : array of shape (n_epochs, n_channels, n_times)
             The epochs data
         """
-        if self.preload:
-            return self._data
-        else:
-            data = self._get_data_from_disk()
-            return data
+        raise NotImplementedError('get_data() must be implemented in derived '
+                                  'class.')
```

Comment:
Since there was no way to read the `else` statement here in a test, since `EpochsArray` has no notion of `preload=False`.  I refactored this a bit. The `get_data` function should now be implemented in all classes inheriting from `_BaseEpochs`.